src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -40,8 +40,11 @@ public override NoPia.EmbeddedTypesManager EmbeddedTypesManagerOpt
         /// </summary>
         private Dictionary<FieldSymbol, NamedTypeSymbol> _fixedImplementationTypes;
 
-        private bool _needsGeneratedIsReadOnlyAttribute_IsFrozen;
         private bool _needsGeneratedIsReadOnlyAttribute_Value;
+        private bool _needsGeneratedIsByRefLikeAttribute_Value;
```

Comment:
>_needsGeneratedIsByRefLikeAttribute_Value [](start = 21, length = 41)

It looks like we never take advantage of this field. Please remove, if that is the case.